main.py

```diff
@@ -18,6 +18,26 @@
     await ctx.send('pong')
 
 
+@bot.command()
+async def listall(ctx):
+    await ctx.send('```--- Commands List --- \n'
+                   '.coach                   - Returns a random motivational quote\n'
+                   '.gifme     {search_term} - Search for a Gif\n'
+                   '.horoscopo {horoscopo}   - Search for you daily horoscope\n'
+                   '.ping                    - Check if bot is Alive\n'
+                   '.youtube   {search_term} - Search for a Youtube Video\n'
+                   '.imageme   {search_term} - Search for an image in Google\n'
+                   '```')
+
+
+@bot.command()
+async def coach(ctx):
+    try:
+        r = requests.get(url="http://api.forismatic.com/api/1.0/?method=getQuote&key=457653&format=json&lang=en")
```

Comment:
hardcoded key here

Reply:
this is not a "key, key" ths parameter defines the randomization of the quotes.